### PR TITLE
Fixes #36253 - delete 36096 leftovers in job_invocation

### PR DESCRIPTION
**Pull request by MariaAga**
sorry :upside_down_face:
removes debugging leftovers from https://github.com/theforeman/foreman_remote_execution/pull/788

### CODE REVIEW

File: app/controllers/ui_job_wizard_controller.rb [hunk 1]
```diff
@@ -68,9 +68,6 @@ def job_invocation
     job_organization = Taxonomy.find_by(id: job.task.input[:current_organization_id])
     job_location = Taxonomy.find_by(id: job.task.input[:current_location_id])
     render :json => {
-      :provider_input_values => composer[:template_invocations][0][:provider_input_values],
```
Comment (adamruzicka) [inline app/controllers/ui_job_wizard_controller.rb:71]: Don't we need at least one copy of the provider input values? Or are they bundled within the regular inputs?

Comment (MariaAga) [reply app/controllers/ui_job_wizard_controller.rb:71]: was added here: https://github.com/theforeman/foreman_remote_execution/pull/788/files#diff-494fb439c4dcbef79b8470f33ba943aa649b2d8e698a9bd26c5353e9e33158f7R22

The keys that were deleted were never used in the js side, and js specified which keys it uses.

**Review by adamruzicka — APPROVED** [review]
Let's get rid of it

**Comment by adamruzicka** [comment]
Thank you @MariaAga !

**Pull request merged**